docs/knowledge_base/tips/multipage_application.md

```diff
@@ -0,0 +1,191 @@
+# Multi-page Application
```

Comment:
The title from the nav bar will be added. We will have 2 titles. So "Multi-page Application" should be the title in the nav bar (`mkdocs.yml_template` file) and we should remove it from this file.
```suggestion
```